### ISSUE TITLE
[$15][MSFT-65] User have to press tab key two time to focus on item

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2780]
**Steps to Reproduce**
1. Navigate to https://www.topcoder.com/members/doremihong
2. Try to mouse hover on primary header menu: Compete or Tracks or Community then click on black background color
3. Use the tab key to focus individual item in sub-menu

**Expected Result**
User only need to press tab key one time to focus on item

**Actual Result**
User have to press tab key two time to focus on item

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: 
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F8

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2780 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2780 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096304 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096304 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu This is happening because of the link inside the list element, inorder to disable that we need to modify the Link element and add tabIndex to that which is inside topcoder-react-utils. What to do now?

**Comment by mishacucicea** [comment]
@asadath1395 This is part of the homepage which is out of scope for now.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097865 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
